source/collections/Vector.ooc

```diff
@@ -121,9 +120,7 @@ HeapVector: class <T> extends Vector<T> {
 
 	operator []= (index: Int, item: T) {
 		version (safe) {
-			// TODO: Should we also check that index < count?
-			// TODO: Should this affect count, if index > count?
-			if (index >= this capacity || index < 0)
+			if (index >= this capacity || index < 0 || index >= this capacity)
```

Comment:
Also: oops. The `TODO`s refer to `count` which doesn't even exist here. I didn't see the whole if-statement once I corrected for that.